### PR TITLE
RB & FF:  Update with changes from 202202 

### PR DESCRIPTION
**Pull request by spbrogan**
## Description

Cherry-pick a few changes that were in 202202 but not in 202208

## How This Was Tested

none

## Integration Instructions

none

### CODE REVIEW

File: MfciPkg/Docs/Mfci_Integration_Guide.md [hunk 1]
```diff
@@ -33,15 +33,57 @@ Add the following to your EDK2 Flash Descriptor File (FDF)
...
+#### Including Modules
+
 MfciPkg provides a ```.dsc.inc``` that can be ```!include``` in your platform DSC.
 An example follows:
 
 ```INI
 !include MfciPkg/MfciPkg.dsc.inc
 ```
 
+Mfci is still considered pre-production, so OPT_INTO_MFCI_PRE_PRODUCTION will need to be set at the top of the Dsc file
```
Comment (spbrogan) [inline MfciPkg/Docs/Mfci_Integration_Guide.md]: Is this a merge conflict?  I thought we removed this text from 202202?  @kuqin12  @apop5

**Review by makubacki — APPROVED** [review]
Did not compare contents, just commit difference between 202202 and 202208 branch.

**Review by kuqin12 — APPROVED** [review]

File: MfciPkg/Docs/Mfci_Integration_Guide.md (hunk 1 above)
Comment (apop5) [reply MfciPkg/Docs/Mfci_Integration_Guide.md]: Yes, I errantly added them back. 

I will put together a PR to drop them again.

Comment (kuqin12) [reply MfciPkg/Docs/Mfci_Integration_Guide.md]: This should not be here.

**Review by apop5 — APPROVED** [review]

**Review by kenlautner — APPROVED** [review]

**Review by mikeytdisco — APPROVED** [review]

**Pull request merged**